File: mkdir"Aula3 - Variaveis Tipos de Dados E Operadores'/calculdora.java
import java.util.Scanner; //clasee para leitura de dados 
import java.util.Locale; //para grantir  o uso de ponto decimal em vez de virgula

public class calculadors {

    public static void main(string[] args){

        Locale.setDefault(Locale.US);

        Scanner sc = new Scanner(System.in);

        System.out.printin(x:"---Clculadora de Negocios (Java) ---");

        System.out.print(s:"Digite o primeiro número:");
        double numero1 = sc.nestDouble();

        System.out.print(s:"Digite o segundo número:");
        double numero2 = sc.nestDouble();

        double soma = numero1 + numero2;
        double subtracao = numero1 - numero2;
        double multiplicacao = numero1 * numero2;
        double divisao = numero1 / numero2;

        System.out.printf(format: "Soma: %.2f%n", soma);
        System.out.printf(format: "Subtração: %.2f%n", subtracao);
        System.out.printf(format: "Multiplicação: %.2f%n", Multiplicacao);
        System.out.printf(format: "Divisão: %.2f%n", divisao);

        sc.close();







    }
}

File: mkdir"Aula3 - Variaveis Tipos de Dados E Operadores'/calculdora.cpp
#include <iostream> // bliblioteca parab entrada e saida de dados//
#include <iomanip> //para fatorar as casas decimais 
using namespace std;

int main() {
    int numero1 = 0, numero2 =0, soma, subtracao , multiplicacao;
    float divisao;

    cout << "Digite o primeiro numero:"; cin >> numero1;
    cout << " Digite o segundo numero"; cin >> numero2;

    soma = numero1 + numero2;
    subtracao = numero1 - numero2;
    multiplicacao = numero1 * numero2;
    divisao = numero1 / numero2;

    cout << fixed << setprecision(2);
    cout << "soma:"<< soma << endl;
    cout << "subtracao:"<< subtracao << endl;
    cout << "multiplicacao:"<< multiplicacao << endl;
    cout << "divisao:"<< divisao << endl;

    return 0;
    
}